src/Install/Installer.php

```diff
@@ -276,6 +279,49 @@ private function awaitingMollieOrderState($languageId)
         return true;
     }
 
+    /**
+     * @param $languageId
+     * @return bool
+     * @throws PrestaShopDatabaseException
+     * @throws PrestaShopException
+     */
+    public function orderCompletedOrderState($languageId)
+    {
+        $stateExists = false;
+        $states = OrderState::getOrderStates((int)$languageId);
+        foreach ($states as $state) {
+            if ($this->module->lang('Completed') === $state['name']) {
```

Comment:
`$this->module->lang('Completed')` - to my understanding this can be a translated value so this condition might not always be true ? :thinking:

Reply:
I mean not always produce result as expected ?